Makefile.ompi-rules

```diff
@@ -21,7 +21,7 @@ endif
 
 .1in.1:
 	@ echo "  BUILDING $@"
-	@ $(top_srcdir)/ompi/mpi/man/make_manpage.pl \
+	$(AM_V_CC) $(top_srcdir)/ompi/mpi/man/make_manpage.pl \
```

Comment:
I think this should be `$(OMPI_V_GEN)`.  Shouldn't the `echo` statement be removed, too?